osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -310,7 +311,7 @@ public static PrivateKey readPrivateKeyFromFile(final String filename) {
         }
     }
 
-    private static KeyFactory getKeyFactory() {
+    private static KeyFactory getKeyFactory() throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)